tests/Aspire.Microsoft.Azure.Cosmos.Tests/ConformanceTests.cs

```diff
@@ -10,10 +11,22 @@
 
 namespace Aspire.Microsoft.Azure.Cosmos.Tests;
 
-public class ConformanceTests : ConformanceTests<CosmosClient, MicrosoftAzureCosmosSettings>
+public class ConformanceTests : ConformanceTests<CosmosClient, MicrosoftAzureCosmosSettings>, IClassFixture<CosmosContainerFixture>
 {
+    private readonly CosmosContainerFixture _containerFixture;
+    private string ConnectionString { get; }
+
+    public ConformanceTests(CosmosContainerFixture containerFixture)
```

Comment:
This should be nullable. See other implementations like `tests/Aspire.Azure.Npgsql.Tests/ConformanceTests.cs`